src/codegen/doc/mod.rs

```diff
@@ -281,8 +281,7 @@ fn create_object_doc(w: &mut dyn Write, env: &Env, info: &analysis::object::Info
             // field could have been renamed.
             if let Some(trait_name) = configured_functions
                 .iter()
-                .filter_map(|f| f.doc_trait_name.as_ref())
-                .next()
+                .find_map(|f| f.doc_trait_name.as_ref())
```

Comment:
We should really start enabling `clippy::pedantic` lints like https://rust-lang.github.io/rust-clippy/master/#filter_map_next, nice catch!

(Or just enable them now and fix the fallout, but there's `62` warnings on stable and `70` warnings on nightly...)